src/window.rs

```diff
@@ -403,3 +415,20 @@ pub unsafe fn decorate_window(view: *mut AnyObject, _position: LogicalPosition<f
             | NSWindowStyleMask::Miniaturizable,
     );
 }
+
+/// Find the config with the maximum number of samples, so our triangle will be
+/// smooth.
+pub fn gl_config_picker(configs: Box<dyn Iterator<Item = Config> + '_>) -> Config {
```

Comment:
Thanks for finding the cause! I didn't know not sharing config would cause error on some platforms.
Could you keep this method in `rendering` module? I hope it can keep most gl methods there.